connectors/src/lib/dust_api.ts

```diff
@@ -203,6 +398,97 @@ export async function processStreamedChatResponse(res: Response) {
   return new Ok({ eventStream: streamEvents() });
 }
 
+/**
+ * This help functions process a streamed response in the format of the Dust API for running
+ * streamed apps.
+ *
+ * @param res an HTTP response ready to be consumed as a stream
+ */
+export async function processStreamedV2ChatResponse(res: Response) {
```

Comment:
Why do we need a new function?

In any case it should be called `processCreateConversationEvents`

Reply:
Can we rename to `processCreateConversationEvents`